Remotion/Web/Development.WebTesting/CoypuElementScopeExtensions.cs

```diff
@@ -248,5 +247,27 @@ public static void Unhover ([NotNull] this ElementScope scope)
 
       Cursor.Position = new Point (0, 0);
     }
+
+    /// <summary>
+    /// Gets an attribute from the scope or throws a <see cref="MissingHtmlException"/> if it could not be found. Wraps <see cref="ElementScope.this"/>.

```

Comment:
Gets the specified HTML attribute's value from the paramref=scope.
Drop rest of summary. Exception info usually not part of summary. the "Wraps..." also doesn't help when reading what the method does.

Reply:
Fixed in https://github.com/re-motion/Trunk-SVN2Git/pull/11/commits/07ec2f7cdccd2a2b833b0c64cd692af7fe4d79f0.